app/web/template/layout/input.gohtml

```diff
@@ -1,6 +1,11 @@
 {{ define "input" }}
   <div class="govuk-form-group {{ if index .top.Errors .name }}govuk-form-group--error{{ end }}">
-    <label class="govuk-label" for="f-{{ .name }}">{{ tr .top.App .label }}</label>
+    <label class="{{ if .boldLabel }}govuk-heading-m{{ else }}govuk-label{{ end }}" for="f-{{ .name }}">{{ tr .top.App .label }}</label>
```

Comment:
I think you could use `if .context` here since that is the only time we'd want the label to be bold

Reply:
Yup, good call.